mapo/agents/mapo/mapo_policy.py

```diff
@@ -149,11 +172,11 @@ def grads_and_vars(loss, optim, variables):
         actor=actor_grads_and_vars,
     )
 
-    policy.grad_stats = {}
-    if config["debug"]:
-        _add_grad_stats(policy.grad_stats, "actor", actor_grads_and_vars)
-        _add_grad_stats(policy.grad_stats, "critic", critic_grads_and_vars)
-        _add_grad_stats(policy.grad_stats, "dynamics", dynamics_grads_and_vars)
+    # policy.grad_stats = {}
+    # if config["debug"]:
```

Comment:
What about adding summaries for the norm of the gradient along side the histograms?

Reply:
Done!